### ISSUE TITLE
Dependency installation fails

### ISSUE THREAD

**Issue by sergei1152** [isaacs/node-glob#172]
Hey it looks like the dev dependencies are failing to install on all the platforms i've tested so far, inlcluding Ubuntu, Windows, and Redhat. It looks to be some issue with running the script `benchclean.sh`. However cloning this repository using git and doing an npm install seems to install everything fine, which is weird since it looks like the latest release on git and the latest release on npm are the same version. 

Here is the console output:

```
nodebot@jtcjs09:~/svt-tool$ npm install glob
npm WARN package.json querystring@0.2.0 querystring is also the name of a node core module.
glob@5.0.0 node_modules/glob
├── inherits@2.0.1
├── once@1.3.1 (wrappy@1.0.1)
├── inflight@1.0.4 (wrappy@1.0.1)
└── minimatch@2.0.1 (brace-expansion@1.1.0)
nodebot@jtcjs09:~/svt-tool$ cd node_modules/glob/
nodebot@jtcjs09:~/svt-tool/node_modules/glob$ npm install
\
> glob@5.0.0 prepublish /home/nodebot/svt-tool/node_modules/glob
> npm run benchclean


> glob@5.0.0 benchclean /home/nodebot/svt-tool/node_modules/glob
> bash benchclean.sh

bash: benchclean.sh: No such file or directory

npm ERR! Linux 3.13.0-37-generic
npm ERR! argv "/home/nodebot/IBMSVT32/bin/node" "/home/nodebot/IBMSVT32/bin/npm" "run" "benchclean"
npm ERR! node v0.12.0
npm ERR! npm  v2.5.1
npm ERR! code ELIFECYCLE
npm ERR! glob@5.0.0 benchclean: `bash benchclean.sh`
npm ERR! Exit status 127
npm ERR!
npm ERR! Failed at the glob@5.0.0 benchclean script 'bash benchclean.sh'.
npm ERR! This is most likely a problem with the glob package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     bash benchclean.sh
npm ERR! You can get their info via:
npm ERR!     npm owner ls glob
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /home/nodebot/svt-tool/node_modules/glob/npm-debug.log

npm ERR! Linux 3.13.0-37-generic
npm ERR! argv "/home/nodebot/IBMSVT32/bin/node" "/home/nodebot/IBMSVT32/bin/npm" "install"
npm ERR! node v0.12.0
npm ERR! npm  v2.5.1
npm ERR! code ELIFECYCLE
npm ERR! glob@5.0.0 prepublish: `npm run benchclean`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the glob@5.0.0 prepublish script 'npm run benchclean'.
npm ERR! This is most likely a problem with the glob package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     npm run benchclean
npm ERR! You can get their info via:
npm ERR!     npm owner ls glob
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /home/nodebot/svt-tool/node_modules/glob/npm-debug.log
nodebot@jtcjs09:~/svt-tool/node_modules/glob$
```

**Comment by heikki** [comment]
https://docs.npmjs.com/files/package.json#files

> The "files" field is an array of files to include in your project

https://github.com/isaacs/node-glob/blob/abdad4c2e6b0ff22850401ff746194183b950da1/package.json#L11-L15

**Comment by isaacs** [comment]
Discussion in https://github.com/isaacs/node-glob/pull/130

If you want to run tests and benchmarks, check out the git repo.

**Comment by haneefkassam** [comment]
I don't understand how this was closed?  the package.json file has a "prepublish" script, which, according to https://docs.npmjs.com/misc/scripts, runs before the package is published AS WELL as when running npm install.  The prepublish calls "npm run benchclean", which then calls "bash benchclean.sh" (thus relying on benchclean.sh).

I'm trying to install plato, which depends on glob.  This shell script is not installed/pulled when doing an "npm install" of glob, meaning every "npm install" of glob will fail.  This has nothing to do with the "files" section of the package.json.

How was this issue closed?  I'm experiencing it on npm v1.3.24 (with node 0.10.25) AND npm v3.10.10 (with node v6.10.2).

@sergei1152 Was this issue ever resolved for you?
